### PR TITLE
#363 fix.

### PR DESCRIPTION
**Pull request by d3nwah**

### CODE REVIEW

**Comment by d3nwah** [comment]
Also some renames, minor cleanups

File: Source/AliveLib/EvilFart.hpp [hunk 1]
```diff
@@ -5,6 +5,13 @@
 
 enum class LevelIds : __int16;
 
+enum class FartStates : __int16
+{
+    e0_IDLE = 0,
```
Comment (paulsapps) [inline Source/AliveLib/EvilFart.hpp:10]: All the other enums are eName_Number

File: Source/AliveLib/FlyingSlig.cpp [hunk 2]
```diff
@@ -908,7 +908,7 @@ void FlyingSlig::Movement_4396E0()
     field_B8_xpos += field_C4_velx;
     field_BC_ypos += field_C8_vely;
 
-    sub_408C40();
+    setActiveCameraDelayedFromDir_408C40();
```
Comment (paulsapps) [inline Source/AliveLib/FlyingSlig.cpp:911]: all the other methods are MethodName not methodName

File: Source/AliveLib/EvilFart.cpp [hunk 3]
```diff
@@ -316,39 +316,38 @@ __int16 EvilFart::VTakeDamage_423B70(BaseGameObject* pFrom)
     return 1;
 }
 
+void EvilFart::vUpdate_REAL_423100()
```
Comment (paulsapps) [inline Source/AliveLib/EvilFart.cpp:319]: can be removed as it was only for testing?

**Pull request merged**